benefits/eligibility/templates/eligibility/confirm.html

```diff
@@ -1,6 +1,6 @@
 {% extends "core/page.html" %}
 
 {% block main_content %}
-  {% include "core/includes/sign-out-link.html" %}
+  <div class="container">{% include "core/includes/previous-page-link.html" %}</div>
```

Comment:
Knowing that this will change, added a `container` wrapper for some simple alignment